cmd/ndm_daemonset/grpc/api.go

```diff
@@ -68,7 +68,11 @@ func Start() {
 
 		// Listen for requests
 		klog.Infof("Starting server at : %v ", Address)
-		grpcServer.Serve(l)
+		err = grpcServer.Serve(l)
+		if err != nil {
+			klog.Errorf("Unable to Serve %v", err)
+			os.Exit(1)
```

Comment:
@RealHarshThakur - should we exit of log error as apiservice is not critical.

Reply:
Yes, I think we should just log.